scripts/protoGen.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+(cd proto; compile-proto-file --proto rls.proto --out ../lib/Fencer/)
+mv lib/Fencer/Rls.hs lib/Fencer/Proto.hs
+sed -i 's/module Rls/module Fencer.Proto/' lib/Fencer/Proto.hs
+sed -i 's/Rls\./Fencer\.Proto\./g' lib/Fencer/Proto.hs
+sed -i '1s/^/{- HLINT ignore -}\n\n/' lib/Fencer/Proto.hs
+DIFF=$(git diff lib/Fencer/Proto.hs | wc -c)
```

Comment:
Let's move these two lines out of this script and into the job itself, then the script would be doing one job instead of two.

Reply:
That's a good point. Done.